packages/api-db-mongodb/src/db/user.ts

```diff
@@ -475,6 +520,57 @@ export class MongoDBUserAdapter implements DBUserAdapter {
     const {_id: outID} = value
     return this.getUserByID(outID)
   }
+
+  async addOAuth2Account({userID, oauth2Account}: UserOAuth2AccountArgs): Promise<OptionalUser> {
+    const user = await this.users.findOne({_id: userID})
+    if (!user) return null
+
+    const accounts: UserOAuth2Account[] = [...user.oauth2Accounts, oauth2Account]
+
+    const {value} = await this.users.findOneAndUpdate(
+      {_id: userID},
+      {
+        $set: {
+          modifiedAt: new Date(),
+          oauth2Accounts: accounts
+        }
+      },
+      {returnOriginal: false}
```

Comment:
I don't understand what this line does - `{returnOriginal: false}`. Can you explain ?

Reply:
It means that the operation should return the updated entry rather than the original one. But as @bishoymelek pointed out this option is deprecated and should be replaced with `returnNewDocument`.